src/libraries/System.Private.CoreLib/generators/EventSourceGenerator.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Generators
+{
+    [Generator]
+    public partial class EventSourceGenerator : ISourceGenerator
+    {
+        public void Initialize(GeneratorInitializationContext context)
```

Comment:
Might consider a comment that shows typical input code and the resulting output code.

Reply:
Added